src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -149,20 +151,40 @@ const RuleEditor = createReactClass({
     return !!error[field];
   },
 
+  handleEnvironmentChange(val) {
+    // If 'All Environments' is selected remove the environment property from the data
+    if (val === 'all') {
+      this.setState(state => {
+        const rule = {...state.rule};
```

Comment:
Can we keep using `let` until we bulk reformat codebase and add a lint rule?

Reply:
I've been using const pretty consistently for new stuff and noticed others have too. I don't think there's a good reason to insist on let now just so we can add a lint rule soon and reformat this line later.